client/starwhale/api/_impl/data_store.py

```diff
@@ -54,22 +57,116 @@ def _check_move(src: str, dest: str) -> bool:
             return False
 
 
-class SwType:
-    def __init__(
-        self, name: str, pa_type: pa.DataType, nbits: int, default_value: Any
-    ) -> None:
+class SwType(ABC):
```

Comment:
`class SwType(ABC):` -> `class SwType(metaclass=ABCMeta)` ?

Reply:
there is no difference. see https://stackoverflow.com/questions/68569239/what-is-the-difference-between-abstractclassmetaclass-abcmeta-and-class-abstra